library/bignum_core.h

```diff
@@ -262,6 +262,21 @@ int mbedtls_mpi_core_write_be( const mbedtls_mpi_uint *A,
                                unsigned char *output,
                                size_t output_length );
 
+/** \brief              Shift a machine integer right by a number of bits.
+ *
+ *                      Shifting by more bits than there are bit positions
+ *                      in \p X is valid and results in setting \p X to 0.
+ *
+ *                      This function's execution time depends on the value
+ *                      of \p count (and of course \p limbs).
```

Comment:
Minor: most of our functions are not fully constant time and we didn't start to document this part of the behaviour yet. Doing so here might give the impression that documenting this side of the behaviour is a requirement and I don't think that we should start doing it just yet.

Reply:
Counterpoint: better record and verify the information now. It's easy to verify this when you've just written or reviewed the code. Documenting it later will be harder since we'll have to re-familiarize ourselves with the code and may be subject to fatigue if we do it in a batch.

Limitation: claims that something is constant-time should have test code that validates the claim. Here, there's only a claim of non-constant-timeness, which needs no validation.